packages/js/src/classic-editor/plugins/seo-title-width.js

```diff
@@ -17,7 +17,7 @@ const registerSeoTitleWidth = () => {
 		namespace,
 		paper => ( {
 			...paper,
-			seoTitleWidth: helpers.measureTextWidth( paper.seoTitle ),
+			seoTitleWidth: helpers.measureTextWidth( paper.seoTitle || paper.titleTemplate ),
```

Comment:
Providing the fallback here with the template is necessary. This is because we don't save the SEO title in the store anymore if it's exactly the same as the template. As the result, `paper.seoTitle` will be empty in this case, and SEO title width assessment will assume that there is no SEO title set.